### ISSUE TITLE
Multiple aggregates per actor system?

### ISSUE THREAD

**Issue by firehooper** [fun-cqrs/fun-cqrs#94]
I believe fun-cqrs only allows one aggregate type per actor system, correct?  Is adding support for multiple aggregate types in the same actor system on the roadmap?  Would you be open to a PR that allow more than one?  If so could you please point me to which files might need reworked.

**Comment by renatocaval** [comment]
There is nothing holding you to have multiple aggregates. Have you tried it?

**Comment by firehooper** [comment]
Sorry for my misunderstanding.  Yes looks like multiple aggregates can be used in the same actor system, but they will all be stored in the same journal / event stream.  Would it be possible have a specific aggregate use a specific journal?

**Comment by renatocaval** [comment]
No, that won't be possible. In general, events from different aggregates are saved on the same journal, because it's the journal of your system. If they are on the same application, they go to the same journal. 

However, Akka does allow you to configure different journals (since 2.5.x I believe). Fun.CQRS won't support it so you will need to provide a PR for it.
